### ISSUE TITLE
Move `examples` to separate repository 

### ISSUE THREAD

**Issue by chrjohn** [quickfix-j/quickfixj#610]

**Comment by david-gibbs-ig** [comment]
Yes I think that will be fine, its dependency on QFJ version or versions will be explicit and I think it can be decoupled from the QFJ version. For example you may not want to have to re-release it all for each minor release of QFJ.

**Comment by david-gibbs-ig** [comment]
It seems that it's possible to keep the (pruned) history of the existing examples. Probably you know this but I have never done it :-) 
https://amandawalkerbrubaker.medium.com/splitting-your-git-repo-while-maintaining-commit-history-35b9f4597514

**Comment by chrjohn** [comment]
@david-gibbs-ig thanks for the link. :+1:  I didn't know it but I wanted to find out if that is possible since it would be good to keep the commit history.

**Comment by chrjohn** [comment]
@david-gibbs-ig 
But I think that I need to increase the version number to the next major number (i.e. 3.0.0), otherwise it will probably cause problems when it's released under the same groupId and artifactId.

https://mvnrepository.com/artifact/org.quickfixj/quickfixj-examples-banzai

Do you agree? Changing the artifactId to something different might be unexpected.

**Comment by david-gibbs-ig** [comment]
> @david-gibbs-ig But I think that I need to increase the version number to the next major number (i.e. 3.0.0), otherwise it will probably cause problems when it's released under the same groupId and artifactId.
> 
> https://mvnrepository.com/artifact/org.quickfixj/quickfixj-examples-banzai
> 
> Do you agree? Changing the artifactId to something different might be unexpected.

Yes I see what you mean. Well 3.0.0 is fine then and should match peoples' expectations, after this it can be versioned independently.
